e2e/integration-test/e2e_metric_test.go

```diff
@@ -39,38 +35,76 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var kMetric map[string][]float64
-var podlists []string
-var pods *v1.PodList
+type TestKeplerMetric struct {
+	KMetric  map[string][]float64
```

Comment:
Since you have used "TestKeplerMetric" struct to encapsulate Kepler specific items, there is no need to use the "KMetric" name, just change to "Metrics" or "MetricMap".